README.md

```diff
@@ -1,10 +1,36 @@
-# Neovim client for julia
+# Neovim client for Julia
 
 [![Build Status](https://travis-ci.org/bfredl/Neovim.jl.svg?branch=master)](https://travis-ci.org/bfredl/Neovim.jl)
 
-This is a neovim api client for julia. It supports embedding a nvim process in julia and conversely acting as a child process to nvim, as well as connecting to an external instance over a socket. It also works as a plugin host. Currently it assumes `nvim` is in `$PATH`
+Neovim.jl is a Neovim API client and plugin host for Julia. It supports:
 
-Simplest way to test the api client is to spawn an embedded instance:
+- Embedding a nvim process in Julia
+- Acting as a child process to nvim
+- Connecting to external instances over a socket
+
+Currently it assumes `nvim` is in `$PATH`.
+
+
+## Requirements
+
+- Julia 1.0
+- Neovim 0.4 <!-- FIXME: is this right? -->
```

Comment:
I'm not sure what the earliest version of Neovim is that this library supports (perhaps @bfredl does?), but at the very least I think this should be `0.4 or later` instead of just `0.4` (or whichever version is the earliest usable), so that it's clear this library is usable on later versions of Neovim/Neovim master.

Also, perhaps the earlier statement `Currently it assumes nvim is in $PATH` could be consolidated into this line, e.g. `Neovim 0.4 or later (this library assumes it is in $PATH)` or something like that.